cmd/cork/create.go

```diff
@@ -123,8 +125,10 @@ func init() {
 	root.AddCommand(createCmd)
 
 	enterCmd.Flags().AddFlagSet(chrootFlags)
+	enterCmd.Flags().BoolVar(&legacy,
+		"legacy", false, "Use old enter implementation")
 	enterCmd.Flags().BoolVar(&experimental,
-		"experimental", false, "Use new enter implementation")
+		"experimental", false, "No-op. do not use. Kept for compatibilty")
```

Comment:
Maybe use `MarkHidden()`?

Reply:
Huh, didn't know that existed. Maybe `MarkDeprecated()` instead. Come to think of it, I think it will techincally break anyone using `cork enter --experimental=false`, but I'm not sure if we care about that?